tests/data_context/abstract_data_context/test_list_datasources.py

```diff
@@ -1,7 +1,8 @@
 from typing import List
 
 from great_expectations import DataContext
-from great_expectations.data_context.types.base import DatasourceConfig
+from great_expectations.data_context import EphemeralDataContext
+from great_expectations.data_context.types.base import DatasourceConfig, DataContextConfig
```

Comment:
I don't think any of the changes in this file are necessary right? Let's revert this back to its original state

Reply:
+1